modules/nf-core/dysgu/meta.yml

```diff
@@ -0,0 +1,78 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "dysgu"
+## TODO nf-core: Add a description of the module and list keywords
+description: Dysgu calls structural variants (SVs) from mapped sequencing reads. It is designed for accurate and efficient detection of structural variations.
+keywords:
+  - structural variants
+  - sv
+  - vcf
+tools:
+  - dysgu:
+      description: Structural variant caller for mapped sequencing data
+      homepage: https://github.com/kcleal/dysgu
+      documentation: https://github.com/kcleal/dysgu/blob/master/README.rst
+      tool_dev_url: https://github.com/kcleal/dysgu
+      doi: "10.1093/nar/gkac039"
+      licence: ["GPL-3.0-or-later"]
+version: "1.6.2"
+
+## TODO nf-core: Add a description of all of the variables used as input
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
```

Comment:
is this mapping representative? does single_end:false make sense her? probably not?

Reply:
HI @famosab yes I agree with you dysgu takes BAM files as input so that line wasn't necessary. Should I remove it?

Reply:
yes, just put id: sample1